Library/Homebrew/dev-cmd/contributions.rb

```diff
@@ -62,8 +66,9 @@ def run
 
         results = {}
         grand_totals = {}
-        from = args.from.presence || Date.today.prev_year.iso8601
-        to = args.to.presence || (Date.today + 1).iso8601
+        quarter = args.quarter.presence || (((Date.today.month - 1) / 3) + 1)
+        from = args.from.presence || quarter_dates[quarter.to_i]&.first || Date.today.prev_year.iso8601
+        to = args.to.presence || quarter_dates[quarter.to_i]&.last || (Date.today + 1).iso8601
```

Comment:
Feels like it'd now be good to clarify what dates are being used in the output (and before the timely queries in case wrong).